src/config/config.yml

```diff
@@ -0,0 +1,43 @@
+invoke:
+  local:
+    params:
+      apiVersion: 2015-03-31
+      endpoint: http://localhost:3000
+    functions:
+      testResults:
+          name: cvs-svc-test-results
+          mock: tests/resources/test-results-response.json
+      techRecords:
+          name: cvs-svc-technical-records
+          mock: tests/resources/tech-records-response.json
+  remote:
+    params:
+      apiVersion: 2015-03-31
+    functions:
+      testResults:
+          name: test-results-${BRANCH}
+      techRecords:
+          name: technical-records-${BRANCH}
+s3:
+  local:
+    endpoint: http://localhost:7000
+    s3ForcePathStyle: true
+  remote: {}
+mot:
+  endpoint: https://9fjfatqw19.execute-api.eu-west-1.amazonaws.com/cvs-dev/CVS
```

Comment:
Please move the endpoint in the secrets.yml file. Same as we did on certificate generation microservice

Reply:
done